internal/backend/db/dbgorm/ownership.go

```diff
@@ -185,17 +186,25 @@ func (gdb *gormdb) withUserEntity(ctx context.Context, entity string) *gorm.DB {
 }
 
 type OwnershipChecker interface {
-	IsUserEntity(db *gorm.DB, user string, ids ...sdktypes.UUID) error
+	EnsureUserAccessToEntities(db *gorm.DB, user string, ids ...sdktypes.UUID) error
```

Comment:
what does this do? 
if it has no ownership (comparing to WithOwnership), so any user has access, am I wrong?